core/dbt/deprecations.py

```diff
@@ -16,6 +16,11 @@ def name(self) -> str:
             'name not implemented for {}'.format(self)
         )
 
+    def track_deprecation_warn(self) -> None:
+        dbt.tracking.track_deprecation_warn({
```

Comment:
I think mypy is mad here because you don't `import dbt.tracking` at the top. The code works as-is for stupid python reasons because you implemented dbt.exceptions, but I'm surprised flake8 accepted this.

Reply:
Heh. Ok cool I'll fix